client/extensions/zoninator/app/controller.js

```diff
@@ -13,7 +13,7 @@ import Settings from '../components/settings';
 
 export const renderTab = ( component ) => ( context ) => {
 	const siteId = getSiteFragment( context.path );
-	const zoneId = context.params.zone;
+	const zoneId = parseInt( context.params.zone );
```

Comment:
What if I entered a URL like `/zoninator/zone/donnapep.wpsandbox.me/abc`? `NaN` would flow down into the `Zone` component. We probably shouldn't let it get that far. Also, it's recommended to always include the second parameter to the `parseInt` function.

Reply:
I used `0` as a fallback here so the behaviour in such case remains consistent with typing an invalid zone ID number.  
I'll leave displaying a proper 'zone not found' message for a separate PR.